composer.json

```diff
@@ -55,7 +55,7 @@
         "utopia-php/database": "0.59.0",
         "utopia-php/domains": "0.5.*",
         "utopia-php/dsn": "0.2.1",
-        "utopia-php/framework": "0.33.*",
+        "utopia-php/framework": "dev-fix-alias-params as 0.33.99",
```

Comment:
Since framework lib needs extensive reviews, we can merge with this in-here.
We can update later when HTTP gets released. Itwill be blocker for merging feat-sites to main